### ISSUE TITLE
Doc: Create documentation for NodeCloud vocabulary

### ISSUE THREAD

**Issue by rajikaimal** [leopardslab/nodecloud#26]
Current documentation doesn't explain the vocabulary of NodeCloud in a convenient manner. New users should be able to use the services without going through extra hassle to find the matching services in cloud providers and NodeCloud APIs.

A documentation page can be created with the vocabulary of current set of APIs in NodeCloud matched with vocabulary of the cloud providers.

**Comment by PratikDhanave** [comment]
how about swagger integration  ?

**Comment by rajikaimal** [comment]
I have no experience using Swagger with a SDK, is it possible to use Swagger for SDK documentation?

**Comment by Pran-Ker** [comment]
@rajikaimal I agree with you on this . How about I add to the documentation , while I am understanding the project .And then , I send it over to you , so you can go through and add points.
